File: api/src/main/java/dev/ulloasp/mlsuite/schema/review/application/dto/CreateSchemaReviewLinkRequest.java
package dev.ulloasp.mlsuite.schema.review.application.dto;

import java.time.OffsetDateTime;
import java.util.List;

import jakarta.validation.constraints.NotEmpty;
import jakarta.validation.constraints.NotNull;

public record CreateSchemaReviewLinkRequest(
        @NotNull Long schemaId,
        @NotNull Long versionId,
        @NotEmpty List<Long> runIds,
        OffsetDateTime expiresAt) {
}

File: frontend/src/api/review/dtos/create-schema-review-link-request.ts
/*
SPDX-License-Identifier: MIT
Copyright (c) 2025 Pablo Ulloa Santin
*/


export type CreateSchemaReviewLinkRequest = {
  schemaId: string;
  versionId: string;
  runIds: string[];
  expiresAt?: string;
};
